BoxContentSDK/BoxContentSDK/Protocols/BOXContentCacheClientProtocol.h

```diff
@@ -292,6 +292,10 @@
 
 #pragma mark - Search
 
+- (void)cacheSearchRequest:(BOXSearchRequest *)request
```

Comment:
Should this match the params of BOXItemArrayCompletionBlock? (i.e. totalCount, range, etc.)